ens/subgraph.yaml

```diff
@@ -119,119 +119,10 @@ dataSources:
         - event: NameRenewed(indexed uint256,uint256)
           handler: handleNameRenewed
   - kind: ethereum/contract
-    name: Resolver2
+    name: Resolver
     network: mainnet
     source:
       abi: Resolver
-      address: '0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41'
```

Comment:
Why is this removed?

Reply:
this is removed coz any domain can have any resolver and we cannot limit to and address, that's why we are listening to all the events of such signature.
that's why we were also doing `domain.resolver = resolver.id` in resolver events.